src/System.Private.CoreLib/src/System/Reflection/RuntimeMethodBody.cs

```diff
@@ -10,11 +11,11 @@ internal sealed class RuntimeMethodBody : MethodBody
     {
         // This class can only be created from inside the EE.
         private RuntimeMethodBody() { }
-        
-        private byte[] _IL;
-        private ExceptionHandlingClause[] _exceptionHandlingClauses;
-        private LocalVariableInfo[] _localVariables;
-        internal MethodBase _methodBase;
+
+        private byte[] _IL = null!;
+        private ExceptionHandlingClause[] _exceptionHandlingClauses = null!;
+        private LocalVariableInfo[] _localVariables = null!;
```

Comment:
The instance of this class only created using native call https://github.com/dotnet/coreclr/blob/4a2654124ecf38300a717db79926334703abeb70/src/vm/runtimehandles.cpp#L2420 where these values looks like set non null and here it passed to non nullable method parameters. So kept them non nullable